game-core/src/test/java/games/strategy/engine/ClientContextIntegrationTest.java

```diff
@@ -18,6 +17,5 @@ void verifyClientContext() {
     assertThat(ClientContext.engineVersion(), notNullValue());
 
     assertThat(ClientContext.getMapDownloadList(), notNullValue());
-    assertThat(ClientContext.getMapDownloadList().isPresent(), is(true));
```

Comment:
Since this is an integration test, should this assertion simply be converted to check that the returned list is not empty?  If so, I would recommend only calling `ClientContext#getMapDownloadList()` once and store the result for use in both assertions.  Otherwise, each call takes a few seconds to actually download the map list, and thus slows down the build unnecessarily.

Actually, if we're just testing for a non-empty collection, the first assertion becomes superfluous, so you could just collapse them down to a single assertion:

```java
assertThat(ClientContext.getMapDownloadList(), is(not(empty())));
```